Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/Cell.cs

```diff
@@ -6,6 +6,8 @@ public static class Cell
 	{
 		public static readonly BindableProperty DefaultBackgroundColorProperty = BindableProperty.Create(nameof(DefaultBackgroundColor), typeof(Color), typeof(Cell), Color.Default);
 
+		public static readonly BindableProperty ContextActionDisplayProperty = BindableProperty.Create(nameof(ContextActionDisplay), typeof(ContextActionDisplay), typeof(Cell), ContextActionDisplay.Text);
```

Comment:
`typeof(Cell)` will refer to this static class; this should use `typeof(FormsElement)`.